scripts/obtain/uninstall/dotnet-uninstall-pkgs.sh

```diff
@@ -31,8 +34,11 @@ remove_dotnet_pkgs
 [ "$?" -ne 0 ] && echo "Failed to remove dotnet packages." && exit 1
 
 echo "Deleting install root - $dotnet_install_root"
-rm -rf $dotnet_install_location
+rm -rf $dotnet_install_root
 rm -f $dotnet_path_file
 
+echo "Deleting NuGet package location"
```

Comment:
@blackdwarf Built shared apps don't have a host dropped app local, so they rely on the muxer being present, this is not true for standalone apps.

Also @tn0502 is right in that if we consider the pkg as a machine wide installation, and the nuget cache as a user local cache, the two things don't exist in the same context. This is especially true for debs, although I'm not sure how true it is for PKGs.